### DISCUSSION TITLE
Copy-paste with rich content for PR and issue titles (suggestion)

### DISCUSSION THREAD

**Discussion by benjaoming** [community/community#44082] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

Hello :wave: 

I have a suggestion for tweaking a workflow that I think is quite common.

## Current practice

When I reference an issue or PR somewhere else, I usually do this:

1) Copy `Title of PR #1234`
2) Insert that in my destination
3) Copy the URL of the PR from the browser's address bar
4) Mark the title in its destination (chat message, email etc)
5) Paste URL directly on the text (if the application is clever enough) or select "insert link" and then paste the URL

## The feature that I'm missing

A clickable text, like:

```
<h1><a href="https://github.com/organization/repo/issue/1234">Title of Issue #1234</a></h1>
```

This way, I can select it in the browser, and my application will understand that I'm pasting rich content.

If I want the old behavior, I can do `SHIFT+CTRL+V` to paste text-only.

## Workaround

I should add that it's possible to get a clickable text from the issue and PR search lists.

The problem is that I'm typically not on that page or may not have navigated to the issue/PR via that page.

That was all :)

**Comment by rileythomp** [comment]
You can sort of achieve this by scrolling down on the issue/PR and then copy the title:

[Copy-paste with rich content for PR and issue titles (suggestion)](https://github.com/orgs/community/discussions/44082#top)

<img width="1112" height="579" alt="Screenshot 2026-02-11 at 9 44 28 AM" src="https://github.com/user-attachments/assets/207d1d6b-7b94-480b-a18b-c67fa6ac7b7f" />

**Reply by danialzivehdadr** [reply]
Ok
